src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
Makes sense :) before a big release, it might be a good idea to do a small release with the floating reference bugfix, just of glib.